### PR TITLE
Implement "Content" object management/step sentences

### PR DESCRIPTION
**Pull request by joaoinacio**
This implements "generic content" object management using the "Content" context trait.

It also introduces a new `EzBaseContext` class without traits, so that each object trait can be used/tested independently; behat.yml is updated accordingly.
(`bin/behat -c path/to/behat.yml -s Content -di` will only list 'Content' related sentences, not steps/sentences for other objects/traits' )

* [x] Implement object creation, removal, editing and existence checks.
* [x] Documentation - Doc Comments are added for sentences, further doc. done after PR #18

### CODE REVIEW

**Comment by joaoinacio** [comment]
Rebased to master, the failing test suggests some form of SPI cache issue with urlAlias - investigating

File: Context/EzBaseContext.php [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+use EzSystems\BehatBundle\Context\Object;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Behat\Symfony2Extension\Context\KernelAwareContext;
+/**
+ * EzContext has all the needed methods and helpers that are globaly used in contexts
+ */
+class EzBaseContext implements KernelAwareContext
+{
+    const DEFAULT_SITEACCESS_NAME = 'behat_site';
```
Comment (bdunogier) [inline Context/EzBaseContext.php]: The current implementation of ezsystems/ezpublish-kernel#223 doesn't use the behat siteaccesses anymore. How big would the consequences be ?

(note that it doesn't run tests with content yet)

Comment (joaoinacio) [inline Context/EzBaseContext.php]: Is that the correct link? If behat env is being used, this should be available.
As for consequences, I'm not really sure tbh - AFAICT there should be no hard breakage, but tests may rely of different/unwanted env config.

On the other hand, this seems a bit limiting - it should be possible to get the default SA from configuration, I believe...

Comment (andrerom) [inline Context/EzBaseContext.php]: Guess he means https://github.com/ezsystems/ezpublish-community/pull/226 and corresponding kernel change.

Comment (bdunogier) [inline Context/EzBaseContext.php]: @andrerom is right. More specifically https://github.com/ezsystems/ezpublish-community/pull/226/files#diff-83339d7064a83a96ef520f01a1bdc577. But it's been passing tests for a while now, and I think it's harmless.

**Comment by andrerom** [comment]
Is this still relevant?

**Comment by bdunogier** [comment]
> Is this still relevant?

+1

**Pull request closed without merging**